works.html

```diff
@@ -0,0 +1,148 @@
+                    <div class="sub2">Back End Dev</div>
+                    <div class="circle"></div>
+                    <div class="sub2">2015</div>
+                </div>
+                <div class="normal">A daily selection of privately personalized reads; no accounts or sign-ups required.</div>
+                <div class="tags">
+                    <div class="tag">
+                        <div class="tagtxt">html</div>
+                    </div>
+                    <div class="tag">
+                        <div class="tagtxt">css</div>
+                    </div>
+                    <div class="tag">
+                        <div class="tagtxt">javaScript</div>
+                    </div>
+                </div>
+                <div class="action">
+
+                    <button type="button" class="button">
+                            See Project
+                        </button>
+
+                </div>
+            </div>
+        </div>
+        <div class="card">
+            <div class="snapshot"><img src="images/card3.svg" alt="tonic"></div>
+            <div class="block">
+                <div class="title">Tonic</div>
+                <div class="subfram">
+                    <div class="sub1">CANOPY</div>
+                    <div class="circle"></div>
+                    <div class="sub2">Back End Dev</div>
+                    <div class="circle"></div>
+                    <div class="sub2">2015</div>
+                </div>
+                <div class="normal">A daily selection of privately personalized reads; no accounts or sign-ups required.</div>
+                <div class="tags">
+                    <div class="tag">
+                        <div class="tagtxt">html</div>
+                    </div>
+                    <div class="tag">
+                        <div class="tagtxt">css</div>
+                    </div>
+                    <div class="tag">
+                        <div class="tagtxt">javaScript</div>
+                    </div>
+                </div>
+                <div class="action">
+
+                    <button type="button" class="button">
+                            See Project
+                        </button>
+
+                </div>
+            </div>
+        </div>
+        <div class="card2">
+            <div class="snapshot"><img src="images/card4.svg" alt="Multi-Post-Stories"></div>
+            <div class="block2">
+                <div class="title2">Multi-Post<br>Stories</div>
+                <div class="subfram">
+                    <div class="sub1">CANOPY</div>
+                    <div class="circle"></div>
+                    <div class="sub2">Back End Dev</div>
+                    <div class="circle"></div>
+                    <div class="sub2">2015</div>
+                </div>
+                <div class="normal">A daily selection of privately personalized reads; no accounts or sign-ups required.</div>
```

Comment:
- [ ]  Please let's change this `<div>` tag into a `<p>` tag for semantic reasons. The requirements of this project demand us to use a `<p>` tag here.